sdk/maps/azure-maps-search/README.md

```diff
@@ -0,0 +1,170 @@
+# Azure Maps Search Package client library for Python
+
+This package contains a Python SDK for Azure Maps Services for Search.
+Read more about Azure Maps Services [here](https://docs.microsoft.com/azure/azure-maps/)
+
+[Source code](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/maps/azure-maps-search) | [API reference documentation](https://docs.microsoft.com/rest/api/maps/search) | [Product documentation](https://docs.microsoft.com/azure/azure-maps/)
+
+## _Disclaimer_
+
+_Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please refer to <https://github.com/Azure/azure-sdk-for-python/issues/20691>_
+
+## Getting started
+
+### Prerequisites
+
+- Python 3.6 or later is required to use this package.
+- A deployed Maps Services resource. You can use the [Azure Portal](https://docs.microsoft.com/azure/azure-maps/quick-demo-map-app) to set it up.
+
+### Install the package
+
+Install the Azure Maps Service Search SDK.
+
+```bash
+pip install --pre azure-maps-search
+```
+
+### Authenticate the client
+
+In order to interact with the Azure Maps Search service, you will need to create an instance of a client.
+A **credential** is necessary to instantiate the client object.
+
+### Create the client with AzureKeyCredential
+
+Once completed, set the value of the subscription key as environment variable:
+`AZURE_SUBSCRIPTION_KEY`.
+
+To use an `AZURE_SUBSCRIPTION_KEY` as the `credential` parameter,
+pass the key as a string into an instance of [AzureKeyCredential][azure-key-credential].
+
+```python
+from azure.core.credentials import AzureKeyCredential
+from azure.maps.search import SearchClient
+
+credential = AzureKeyCredential(os.environ.get("AZURE_SUBSCRIPTION_KEY"))
+
+search_client = SearchClient(
+    credential=credential,
+)
+```
+
+### Create the client with an Azure Active Directory credential
+
+To use an [Azure Active Directory (AAD) token credential][maps_authentication_aad],
+provide an instance of the desired credential type obtained from the
+[azure-identity][azure_identity_credentials] library.
```

Comment:
Invalid link

Reply:
All the links should be working

Reply:
It's not. Please refer to the page: https://github.com/alextts627/azure-sdk-for-python/tree/maps/onboarding/search-sdk/sdk/maps/azure-maps-search

Reply:
updated